lib/src/draggable_scrollbar.dart

```diff
@@ -27,6 +28,7 @@ class DraggableScrollbar extends StatefulWidget {
     this.currentFirstIndex = 0,
     required this.scrollThumbBuilder,
     this.onChange,
+    required this.scrollDirection
```

Comment:
I'd actually prefer this to be not required, defaulting to `vertical`, so that we don't introduce an unnecessary breaking change.

Reply:
Okay.
Changed it.